porting/common-system-build-errors.rst

```diff
@@ -14,6 +14,19 @@ All APKs and Java libraries should be removed from the Makefiles in ``vendor/[ma
 
 Also check the vendor folders of your dependencies.
 
+Undefined or missing bluetooth headers
+-------------------------------------
+
+.. code-block:: guess
+
+    *TODO: example*
```

Comment:
The example should actually be here before we merge

Reply:
If someone had a nicely log...

Reply:
@Lyuu17 ping